### PR TITLE
Extending .gitignore to ignore generated tests

### PR DESCRIPTION
**Pull request by EricLeFort**
# Description

Adds a .gitignore line for the generated tests since they are quite distracting when working through diffs.

Since the workflow still needs those files, I also added a line to force add them there.

# Tests

N/A, no test changes. I will monitor the dashboard after merging to ensure nothing has been broken.

**Instruction and/or command lines to reproduce your tests:** N/A

**List links for your tests (use go/shortn-gen for any internal link):** N/A

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [ X ] I have performed a self-review of my code.
- [ X ] I have necessary comments in my code, particularly in hard-to-understand areas.
- [ X ] I have run one-shot tests and provided workload links above if applicable. 
- [ X ] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

**Comment by will-cromar** [comment]
Can you update this action too? https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/.github/workflows/generate.yml

**Comment by EricLeFort** [comment]
Wasn't sure if there was a way to get that pre-built action to do what we needed so made it a custom run similar to `diff.yml`.

File: .github/workflows/generate.yml [hunk 1]
```diff
@@ -33,8 +33,9 @@ jobs:
...
-      uses: actions-x/commit@v5
-      with:
-        branch: gen
-        files: k8s/
-        force: true
+      run: |
+        git add --force -N k8s/*/gen/*.yaml
+        git add -N k8s/
+        git commit -m "Automatically updated using GitHub Actions"
+        git push -f
```
Comment (will-cromar) [inline .github/workflows/generate.yml:40]: Should this be `push -f origin gen`? I don't think the upstream is set here.

Comment (EricLeFort) [reply .github/workflows/generate.yml:40]: That's safer regardless, will change.

**Review by will-cromar — APPROVED** [review]
Thanks!

**Pull request merged**